### PR TITLE
Update Netty to the latest version

### PR DESCRIPTION
**Pull request by danotorrey**
Update Netty dependency to the latest version. 

This was recently done in https://github.com/Graylog2/gelfclient/pull/52, so the same update is being done in the graylog2-server core while we're looking at this.

### CODE REVIEW

**Comment by danotorrey** [comment]
A quick smoke-test of the application was successful.

File: pom.xml [hunk 1]
```diff
@@ -135,7 +135,7 @@
         <mongodb-driver.version>3.12.1</mongodb-driver.version>
         <mongojack.version>2.10.1</mongojack.version>
         <natty.version>0.13</natty.version>
-        <netty.version>4.1.52.Final</netty.version>
+        <netty.version>4.1.59.Final</netty.version>
         <netty-tcnative-boringssl-static.version>2.0.34.Final</netty-tcnative-boringssl-static.version>
```
Comment (mpfz0r) [inline pom.xml:139]: Would you mind als updating netty-tcnative to `2.0.36`? 
We've had problems with that in the past, so it's a good practice to update them together

Comment (mpfz0r) [reply pom.xml:139]: For reference: https://github.com/netty/netty-tcnative/releases

Comment (danotorrey) [reply pom.xml]: Thanks for the suggestion. Done.

**Review by bernd — CHANGES_REQUESTED** [review]
Netty 4.1.60 just got released including a fix for a possible security problem.

See: https://netty.io/news/2021/03/09/4-1-60-Final.html

**Review by mpfz0r — APPROVED** [review]
Works for me. Tested HTTPS and TLS enabled beats input

**Comment by mpfz0r** [comment]
@bernd can you approve or remove your change request?

**Review by bernd — APPROVED** [review]

**Pull request merged**